plot_bp_2d.m

```diff
@@ -57,42 +67,58 @@ function plot_bp_2d(handles)
 azqm = azq;
 azqm(isnan(vq_norm)) = NaN;
 
-axes(handles.axes_bp);
-cla(handles.axes_bp,'reset');
-axesm eckert4;
-framem('fedgecolor',200*ones(1,3)/255,'flonlimit',[-120 120]);
-gridm('gcolor',190*ones(1,3)/255,'glinestyle','-');
-axis off
-if get(handles.checkbox_norm,'Value')==0  % if "normalized" not checked
-    geoshow(elqm/pi*180,azqm/pi*180,vq,'displaytype','texturemap');
-    cc = [gui_op.caxis_raw_current(1) gui_op.caxis_raw_current(2)];
-else
-    geoshow(elqm/pi*180,azqm/pi*180,vq_norm,'displaytype','texturemap');
-    cc = [gui_op.caxis_norm_current(1) gui_op.caxis_norm_current(2)];
+if get(handles.show_bp_plot,'Value')
+  if isempty(handles.axes_bp.Children)
+    axes(handles.axes_bp);
+    cla(handles.axes_bp,'reset');
+    axesm eckert4;
+  else
+    axes(handles.axes_bp);
+    cla(handles.axes_bp)
+  end
+  framem('fedgecolor',200*ones(1,3)/255,'flonlimit',[-120 120]);
+  gridm('gcolor',190*ones(1,3)/255,'glinestyle','-');
+  axis off
+  if get(handles.checkbox_norm,'Value')==0  % if "normalized" not checked
+      geoshow(elqm/pi*180,azqm/pi*180,vq,'displaytype','texturemap');

```

Comment:
Should update these to plotting projected bp intensity directly using projected x,y positions (from az,el), since geoshow is much slower than contourf